packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx

```diff
@@ -1,6 +1,339 @@
 ---
 title: Livestream Tutorial
-description: How to build a livestream experience using Stream's video SDKs
+description: How to build a live stream experience using Stream's React Video SDKs
 ---
 
-TODO
+## Quickstart
+In this quickstart, we'll build a live streaming experience that's similar to Twitch.
+
+- The live stream will run on Stream's Edge network of servers around the world
+- You can use ultra-low latency (WebRTC) based live streaming or HLS (slight delay, but better buffering)
+- We'll show how to publish to the live stream from your browser
+- Stream enables you to scale to millions of viewers
+- You can have multiple active participants in the live stream
+
+The app that we are going to build would work in two modes: Host (presenter) and Viewer (audience).
+
+The host will be able to publish the live stream and the viewers will be able to watch the live stream.
+If you want to see a fully-fledged example, check out this [sample app.](https://github.com/GetStream/stream-video-js/tree/main/sample-apps/react/livestream-app)
+
+Let's get started.
+
+## Step 1 - Preparations
+
+Before going into the code, let's make sure we have everything we need.
+
+- Add Stream's React Video SDK to your project:
+  - `yarn install @stream-io/video-react-sdk` or
+  - `npm install @stream-io/video-react-sdk`
+- Setup a Stream account at [GetStream's Dashboard](https://dashboard.getstream.io) or use your existing one
+  - We will need your API key and secret
+
+## Step 2 - Setup the host mode
+
+Hosts should be communicating with each other in real-time. For that purpose, we would use the standard Stream Video Call component, based on WebRTC.
+
+:::note
+In this tutorial, we will skip the details on how a `StreamVideoClient` can be set up. You can read more about it on the following page:
+[Authenticating your Users](../../guides/01-authentication/)
+:::
+
+In the snippet below, we create a connection to the Stream Edge server for the live stream host, we join the call and we publish our audio and video to the live stream.
+In this example, we use the predefined `<PaginatedGridLayout>` for the live stream.
+
+In case you want to have a more custom layout, you can build your own. See [Building Custom Layouts.](../ui-cookbook/participant-list/)
```

Comment:
```suggestion
In case you want to have a more custom layout, you can build your own. See our article about [Building Custom Layouts.](../ui-cookbook/participant-list/)
```

Reply:
```suggestion
If you prefer a more customized layout, you can build your own. See our article about [Building Custom Layouts.](../ui-cookbook/participant-list/)
```